config/application.rb

```diff
@@ -66,6 +66,18 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :delayed_job
 
+    config.conference = {
+      events_per_page: 3,
+      default_logo:    'snapcon_logo.png',
+      default_color:   '#0B3559'
+    }
+
+    config.mailbot = {
+      bcc_address:    'messages@snap.berkeley.edu',
```

Comment:
This definitely should be an environment variable. `OSEM_MESSAGE_BCC_ADDRESS`, perhaps. :)